cogs/badges.py

```diff
@@ -143,7 +143,7 @@ def get_user_badges(self, discord_id):
         ):
             userBadges["achievements"].append(self.discordXGithubBadge)
 
-        discordMemberData = SupabaseClient().read(
+        discordMemberData = PostgresClient().read(
```

Comment:
Can we create 1 class level instance of postgresclient please? Creating a new one each time seems unncessary.